lib/deprecation_tracker.rb

```diff
@@ -18,7 +18,7 @@ def self.callbacks
       @callbacks ||= []
     end
 
-    def warn(*messages, uplevel: nil, category: nil)
+    def warn(*messages, uplevel: nil, category: nil, **kwargs)
```

Comment:
This method override now explicitly accepts both known (`uplevel`, `category`) and unknown (`**kwargs`) keyword arguments. This makes it compatible with upstream libraries (like `sass-embedded`) that may pass unexpected keys.